python/ray/rllib/evaluation/postprocessing.py

```diff
@@ -44,6 +44,7 @@ def compute_advantages(rollout, last_r, gamma, lambda_=1.0, use_gae=True):
         rewards_plus_v = np.concatenate(
             [rollout["rewards"], np.array([last_r])])
         traj["advantages"] = discount(rewards_plus_v, gamma)[:-1]
+        traj["value_targets"] = np.zeros_like(traj["advantages"])
```

Comment:
You can use a critic without using GAE, but this does not allow that functionality. Can you add documentation noting this?

Reply:
Done